crypto/fipsmodule/sha/asm/keccak1600-armv8.pl

```diff
@@ -61,41 +61,28 @@
 
 # $output is the last argument if it looks like a file (it has an extension)
 # $flavour is the first argument if it doesn't look like a file
+$output = $#ARGV >= 0 && $ARGV[$#ARGV] =~ m|\.\w+$| ? pop : undef;
+$flavour = $#ARGV >= 0 && $ARGV[0] !~ m|\.| ? shift : undef;
 
+$0 =~ m/(.*[\/\\])[^\/\\]+$/; $dir=$1;
+( $xlate="${dir}arm-xlate.pl" and -f $xlate ) or
+( $xlate="${dir}../../../perlasm/arm-xlate.pl" and -f $xlate) or
+die "can't locate arm-xlate.pl";
 
-#File keccak1600-armv8.pl is imported from OpenSSL.
-#https://github.com/openssl/openssl/blob/479b9adb88b9050186c1e9fc94879906f378b14b/crypto/sha/asm/keccak1600-armv8.pl
+open OUT,"| \"$^X\" $xlate $flavour \"$output\""
+    or die "can't call $xlate: $!";
```

Comment:
I think it would be best to match what we have in other armv8 .pl files, e.g. https://github.com/awslabs/aws-lc/blob/76905dab6020199d1b28c75762684e7fb10c3c5b/crypto/cipher_extra/asm/chacha20_poly1305_armv8.pl#L23-L36 and
https://github.com/awslabs/aws-lc/blob/76905dab6020199d1b28c75762684e7fb10c3c5b/crypto/fipsmodule/aes/asm/aesv8-armx.pl#L42-L54

Reply:
I changed it back. Thanks.